src/components/SpecialResolution/Rules.vue

```diff
@@ -0,0 +1,559 @@
+                  x-small
+                  label
+                  color="primary"
+                  text-color="white"
+                >
+                  {{ getEditedLabel }}
+                </v-chip>
+              </v-col>
+            </v-col>
+
+            <!-- Display Mode -->
+            <template v-if="!hasChanged">
+              <v-col
+                :cols="isEditing ? 10 : 9"
+                class="pr-2"
+              >
+                <div
+                  v-if="!isEditing"
+                  class="mx-4"
+                >
+                  <div
+                    v-if="!getRules || !getRules.key"
+                  >
+                    <span
+                      id="rules-paper-not-changed"
+                      class="ml-7 d-block"
+                    >
+                      Available on paper only
+                    </span>
+                  </div>
+                  <div
+                    v-else
+                  >
+                    <div
+                      :key="getRules.key"
+                      class="download-link-container"
+                    >
+                      <v-icon
+                        color="primary"
+                        class="mt-n1"
+                      >
+                        mdi-file-pdf-outline
+                      </v-icon>
+                      <a
+                        download
+                        class="ml-1"
+                        @click="openPdf()"
+                      >
+                        {{ getRules.name }}
+                      </a>
+                    </div>
+                  </div>
+                  <span
+                    v-if="lastUploadedDetails"
+                    class="mt-1 mb-2 last-modified-details"
+                  >
+                    {{ lastUploadedDetails }}
+                  </span>
+                </div>
+                <div v-else>
+                  <div
+                    class="px-4 pt-0 section-container"
+                    :class="{'invalid-text': rulesEditingInvalid}"
+                  >
+                    You can update the rules of association in one of the following ways:
+                  </div>
+                  <v-divider class="mx-4" />
+                  <section
+                    class="px-4 py-2 section-container"
+                  >
+                    <v-btn
+                      id="btn-describe-rules"
+                      text
+                      block
+                      color="primary"
+                      class="pl-0 ml-0"
+                      style="font-size: 1rem"
+                      @click="rulesInResolution = false; rulesInUpload = false;
+                              uploadDropdown = false; describeDropdown = !describeDropdown"
+                    >
+                      <v-expand-transition>
+                        <span :style="describeDropdown ? 'color: black !important; font-weight: bold !important' : ''">
```

Comment:
Prefer to not see CSS styles in the template. Can you conditionally add a classname and then add classes with the styles?